src/synthesizer/abundances/elements.py

```diff
@@ -0,0 +1,142 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Elements:
+
+    """
+    A simple class containing various useful lists and dictionaries.
```

Comment:
This is a very uninformative docstring, could this be updated to describe what it actually contains?